src/PositionManager.sol

```diff
@@ -412,25 +395,40 @@ contract PositionManager is
         }
     }
 
+    /// @notice an internal helper used by Notifier
+    function _setSubscribe(uint256 tokenId) internal override {
+        positionInfo[tokenId] = positionInfo[tokenId].setSubscribe();
+    }
+
+    /// @notice an internal helper used by Notifier
+    function _setUnsubscribe(uint256 tokenId) internal override {
+        positionInfo[tokenId] = positionInfo[tokenId].setUnsubscribe();
+    }
```

Comment:
mini nit: should we rename this to `_setSubscribed` and `_setUnsubscribed` just to be clearer. Feel like subscribe/subscriber is quite overloaded